package.json

```diff
@@ -63,6 +61,7 @@
     "request": "^2.81.0"
   },
   "dependencies": {
+    "argon2": "^0.15.0",
```

Comment:
Why choose `argon2` rather than `scrypt`, `blowfish`, or just many rounds of `SHA-256`?